### PR TITLE
Feat: Add README instructions and eslint

### PR DESCRIPTION
**Pull request by genelorenzSarmiento0408**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+**To check the style if it follows the style guide**:
+
+```cmd
+npm run lint
+```
+
+**To start it in development**:
+
+```cmd
+npm run dev
```
Comment (sukyungdev) [inline README.md:22]: Do I enter this command before I start the development?

**Review by sukyungdev — COMMENTED** [review]
Question about README.md Instructions

**Review by sukyungdev — APPROVED** [review]
Approve Pull Request : Project Setting

**Review by ha-anna — APPROVED** [review]
All seems good for now!

File: README.md (hunk 1 above)
Comment (genelorenzSarmiento0408) [reply README.md:22]: I meant to start the program in development. will change!

**Pull request merged**